web/ui/react-app/src/App.css

```diff
@@ -108,10 +108,11 @@ button.execute-btn {
 
 .graph-controls .time-input input {
     border-right: none;
+    padding-right: 0;
 }
 
 div.time-input {
-    width: 240px !important;
+    width: 250px !important;
```

Comment:
This needs to be a bit wider, in the Table view, the seconds are a bit cut off.

Reply:
Also, the "X" timestamp removal button is differently styled (solid vs. transparent) in the Table view than the Graph view (via `.graph-controls .clear-time-btn` in `App.css`). We should probably change both views to be transparent.

Reply:
(also, other sizings seem a bit off between the two time pickers - would be great to get this consistent and cleaned up)